app/views/users/index.html.erb

```diff
@@ -0,0 +1,15 @@
+
+<h1>Users</h1>
+<div class="user-container">
+<% @users.each do |user| %>
+  <div class="user-card">
+    <div class="user-image">
+          <img src="<%= user.photo_link %>" alt="<%= user.name %> photo" class="img-fluid rounded-circle">
+      </div>
+    <div class="user-info">
+      <h3><%= link_to user.name, user_path(user) %></h3>
+      <p class="comment">Number of posts: <%= user.posts.count %></p>
+    </div>
+  </div>
+<% end %>
+</div>
```

Comment:
- [ ] [REQUIRED] Good job so far! I appreciate what you have done and you should pay more attention on the function as styling is nice to have. The `index` view page should match the given [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png) 🙏🏼
- [NOTE] I can only access to see this server `http://127.0.0.1:3000/users` therefore if other pages also same like this please fix them too 🙏🏼 

![image](https://github.com/hydaline-code/myblogapp/assets/63915024/2fe7e341-c9b0-48e9-889a-e23644181bad)